src/test/java/edu/pdx/cs/ConnectionInfoTest.java

```diff
@@ -0,0 +1,86 @@
+package edu.pdx.cs;
+
+import org.junit.Test;
+
+import java.net.UnknownHostException;
+import java.nio.file.Paths;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+import static org.hamcrest.MatcherAssert.assertThat;
+
+
+public class ConnectionInfoTest extends InvokeMainTestCase{
+
+
+    private InvokeMainTestCase. MainMethodResult invokeMain(String... args) {
+        return invokeMain(App.class, args);
+    };
```

Comment:
Remove this code and the `extends InvokeMainTestCase` part of class declaration.

An atomic test case for a getter might look something like the following (I'm bad at test names)
```
@Test
public void getPortReturnsExpected() { 
ConnectionInfo c = new ConnectionInfo("host", 22, "aang");
assertThat(c.getPort(), equalTo(22));
}
```